webapp/src/app.tsx

```diff
@@ -869,7 +886,10 @@ export class ProjectView
 
     openHome() {
         pxt.tickEvent("menu.open");
-        this.home.showHome();
+        this.stopSimulator();
+        this.setState({ home: true });
+        // clear the hash
+        pxt.BrowserUtils.changeHash("", true);
```

Comment:
we should change the hash, then the onhashchanged handler should actually do the state transition